lib/client.js

```diff
@@ -557,7 +589,7 @@ Client.prototype.getMoneyInChequeDetails  = function (ip, _opts) {
     updateDate: utils.dateToUnix(_opts.from) || 0
   });
 
-  return this._request('GetMoneyInChequeDetails', ip, opts).get('TRANS').get('HPAY');
+  return this._request('GetMoneyInChequeDetails', ip, opts).then(this._extractTransactionList);
```

Comment:
```suggestion
  return this._request('GetMoneyInChequeDetails', ip, opts).then(this._extractTransaction);
```

Reply:
+ check method `getMoneyInChequeDetails` in `transaction-factory` and you'll see it using a `.map` method

Reply:
My bad.